src/main/java/juja/sqlcmd/DatabaseManager.java

```diff
@@ -0,0 +1,52 @@
+package juja.sqlcmd;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class DatabaseManager {
+    private static final String DB_CONNECTION_URL = "jdbc:postgresql://127.0.0.1:5432/";
+    private static final String POSTGRESQL_DRIVER = "org.postgresql.Driver";
+    private Connection connection;
+
+    public boolean connect(String databaseName, String user, String password) {
+        try {
+            Class.forName(POSTGRESQL_DRIVER);
+            connection = DriverManager.getConnection(DB_CONNECTION_URL + databaseName, user, password);
+            return true;
+        } catch (ClassNotFoundException e) {
+            System.out.println("Please add JDBC jar to your project");
+            return false;
+        } catch (SQLException e) {
+            System.out.println(String.format("Can not connect to %s with user = %s and password %s", databaseName, user, password));
+            return false;
+        }
+    }
+
+    public String[] getTableNames() throws SQLException {
+        String sqlQuery = "SELECT table_name FROM information_schema.tables WHERE table_schema='public' AND table_type='BASE TABLE'";
+        try (PreparedStatement preparedStatement = connection.prepareStatement(sqlQuery, ResultSet.TYPE_SCROLL_SENSITIVE,
+                ResultSet.CONCUR_UPDATABLE); ResultSet resultSet = preparedStatement.executeQuery()) {
+            int tableSize = countTable(resultSet);
```

Comment:
I think it's better to declare ResultSet on another line. You can write it like

`try (PreparedStatement preparedStatement = connection.prepareStatement(
                sqlQuery, ResultSet.TYPE_SCROLL_SENSITIVE, ResultSet.CONCUR_UPDATABLE);
             ResultSet resultSet = preparedStatement.executeQuery()) {`